packages/entities/entities-plugins/src/components/free-form/shared/RadioField.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <!-- missing schema alert -->
+  <KAlert
+    v-if="field.error"
+    appearance="danger"
+    :message="field.error.message"
+  />
+
+  <div v-else>
+    <KLabel v-bind="fieldAttrs.labelAttributes">
+      {{ fieldAttrs.label }}
+      <template
+        v-if="fieldAttrs.labelAttributes?.info"
+        #tooltip
+      >
+        <!-- eslint-disable-next-line vue/no-v-html -->
+        <div v-html="fieldAttrs.labelAttributes.info" />
+      </template>
+    </KLabel>
```

Comment:
I am wondering should we put conditional rendering here for props without label, since we have the similar issue with `NumberField` which pass `label=""` to the component to opt out label, maybe it is better for us to opt out label when `label` is `undefined` or `null`.

Reply:
Yeah good point, we can consider adding this feature during the Label refactoring [KM-1308](https://konghq.atlassian.net/browse/KM-1308)